aspnet/mvc/models/validation/sample/User.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.AspNet.Mvc;
+using System;
+
+namespace MVCMovie.Models
+{
+public class UserRepository : IUserRepository
```

Comment:
Are this class and `UsersController` purposefully outdented? Why are the remaining classes indented more normally if yes?